helix-view/src/input.rs

```diff
@@ -1,11 +1,26 @@
 //! Input event handling, currently backed by crossterm.
 use anyhow::{anyhow, Error};
 use helix_core::unicode::width::UnicodeWidthStr;
-use serde::de::{self, Deserialize, Deserializer};
+use serde::de::{Deserialize, Deserializer};
 use std::fmt;
+use std::sync::atomic::{AtomicBool, Ordering};
 
 use crate::keyboard::{KeyCode, KeyModifiers};
 
+static HAD_CONFIG_ERROR: AtomicBool = AtomicBool::new(false);
```

Comment:
I think doing global variable for this will make stuff hard later, doing it as variable passing in rust is more common and less error prone.